pilot/pkg/model/context.go

```diff
@@ -273,6 +273,15 @@ type Proxy struct {
 	// NOTE: DO NOT USE THIS FIELD TO CONSTRUCT DNS NAMES
 	ConfigNamespace string
 
+	// Labels specifies the set of workload instance (ex: k8s pod) labels associated with this node.
+	// Labels can be different from that in Metadata because of pod labels update after startup,
+	// while NodeMetadata.Labels are set during bootstrap.
+	Labels map[string]string
+
+	// IstioMetaLabels contains the labels specified by ISTIO_METAJSON_LABELS and platform instance,
```

Comment:
Is it "ISTIO_METAJSON_LABELS and platform instance" or "ISTIO_METAJSON_LABELS **minus** platform instance"?

Reply:
platform instance labels only from GCP vm.

For other envs, it is only that in ISTIO_METAJSON_LABELS